provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerTransformer.java

```diff
@@ -49,25 +51,34 @@ Address address(ProviderResponse.Value value) {
 
   @Override
   public Practitioner apply(PractitionerWrapper ppmsData) {
-    ProviderResponse.Value response = ppmsData.providerResponse().value().get(0);
-    ProviderContactsResponse.Value contacts;
-    if (ppmsData.providerContactsResponse().value().size() > 0) {
-      contacts = ppmsData.providerContactsResponse().value().get(0);
-    } else {
-      contacts = null;
-    }
+    ProviderResponse.Value providerResponse = ppmsData.providerResponse().value().get(0);
+    ProviderContactsResponse.Value providerContacts =
+        ppmsData.providerContactsResponse().value() == null
+                || ppmsData.providerContactsResponse().value().isEmpty()
+            ? null
+            : ppmsData.providerContactsResponse().value().get(0);
+    ProviderServicesResponse.Value providerServices =
+        ppmsData.providerServicesResponse().value() == null
+                || ppmsData.providerServicesResponse().value().isEmpty()
+            ? null
+            : ppmsData.providerServicesResponse().value().get(0);
     List<Practitioner.PractitionerIdentifier> identifiers = new ArrayList<>();
-    identifiers.add(identifier(response));
+    identifiers.add(identifier(providerResponse));
     return Practitioner.builder()
         .resourceType("Practitioner")
-        .active(active(response.providerStatusReason()))
-        .id(response.providerIdentifier().toString())
+        .active(active(providerResponse.providerStatusReason()))
+        .id(providerResponse.providerIdentifier().toString())
         .identifier(identifiers)
-        .name(name(response.name()))
-        .gender(gender(response.providerGender()))
-        .address(addresses(response))
-        .birthDate((contacts == null) ? null : contacts.birthday())
-        .telecom(telecoms(contacts))
+        .name(name(providerResponse.name()))
+        .gender(gender(providerResponse.providerGender()))
+        .address(addresses(providerResponse))
+        .birthDate((providerContacts == null) ? null : providerContacts.birthday())
+        .telecom(
+            providerServicesTelecoms(providerServices) != null
+                ? providerServicesTelecoms(providerServices)
+                : providerTelecoms(providerResponse) != null
+                    ? providerTelecoms(providerResponse)
+                    : providerContactsTelecom(providerContacts))
```

Comment:
The doubly-nested ternary is hard to understand... let's extract this to its own method and replace the ternaries with real `if` blocks to make the logic easier to follow.